src/open_inwoner/cms/products/cms_plugins.py

```diff
@@ -77,6 +77,9 @@ def render(self, context, instance, placeholder):
                 config.enable_categories_filtering_with_zaken
                 and request.user.is_authenticated
                 and (request.user.bsn or request.user.kvk)
+                and
+                # There might be no ZGW backend configured
+                ZGWApiGroupConfig.objects.count()
```

Comment:
```suggestion
                ZGWApiGroupConfig.objects.exists()
```
More efficient query

Reply:
You're right!